network-api/networkapi/buyersguide/models.py

```diff
@@ -341,6 +358,7 @@ class Product(models.Model):
             heading="Publication status"
         ),
         MultiFieldPanel([
+            FieldPanel('not_safe_for_work'),
```

Comment:
This feels like the wrong label entirely, especially if we're using this to mark items as being "adult items" and have a help text that says that checking this box blurs the item and adds an 18+ label. These items may not be safe for work, but they're certainly also not safe for home if you're a 16yo kid and your parents are looking the screen. NSFW is a convenient piece of internet jargon, but it also feels like it distorts why we're adding this feature. If it was _just_ not safe for work, it shouldn't also be 18+ by default.

(and this is absolutely a pet peeve =)

/cc @caltrider @kristinashu

Reply:
I can change it to something else. What label name should we use? Would any of the follow works? 
- `adult item`,
- `adult content`, 
- or `18 plus` (`eighteen plus`? since I don't know if we want the field name to start with number)

Reply:
`adult content` it is then!